src/util.c

```diff
@@ -24,11 +24,6 @@
 #include "container.h"
 #include "../config.h"
 
-char *read_cmdline(void)
```

Comment:
I'm not sure we should remove it, we may pass the debug level from it in future.
#170